spec/serializers/provider_serializer_spec.rb

```diff
@@ -57,7 +57,25 @@
     it { should eq provider.ucas_preferences.send_application_alerts_before_type_cast }
   end
 
-  context "when UCAS preferences are missing" do
+  describe 'application alert recipient' do
+    context 'if not nil the application alert recipient is serialized upon instantiation of the contact object' do
+      subject { serialize(provider)['contacts'].find { |c| c[:type] == 'application_alert_recipient' } }
```

Comment:
It might be good to break this up a bit? Readability feels like it could be better perhaps.
```
let(:application_alert_recipient) do
  serialize(provider)['contacts'].find { |contact| contact[:type] == 'application_alert_recipient' }
end

subject { application_alert_recipient }
```

Reply:
Have actioned both your changes. Thanks for your feedback Heidar 👍